lib/database_connection.rb

```diff
@@ -0,0 +1,15 @@
+require 'pg'
+
+class DatabaseConnection 
+  def self.setup(database)
+    @connection = PG.connect(dbname: database)
+  end
+
+  def self.connection
+    @connection
+  end
+
+  def self.query(query_string)
+    @connection.exec(query_string)
```

Comment:
what would happen if a user sent a peep with a ' or a ; in it? Message.self just interpolates the message into the SQL string, then DatabaseConnection.query would fail because the sql string is invalid.

Reply:
You can use connection.exec_params instead, which can interpolate data into placeholders for you and avoid this error